### ISSUE TITLE
Returning null value from worker returns empty object instead on front

### ISSUE THREAD

**Issue by aurelien-baudet** [zetapush/zetapush#182]
**I'm submitting a ...** (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
```

**Scope** (check one with "x")
```
[ ] <root>
[ ] @zetapush/cli
[x] @zetapush/client
[ ] @zetapush/cometd
[ ] @zetapush/common
[ ] @zetapush/core
[ ] @zetapush/create
[ ] @zetapush/http-server
[ ] @zetapush/platform-legacy
[ ] @zetapush/testing
[ ] @zetapush/troubleshooting
[ ] @zetapush/user-management
[x] @zetapush/worker
```

**Current behavior**

Calling a method defined in a worker that returns null, the client received an empty object instead of null.

**Expected behavior**

If the API returns null, the client should also receive null.

**Minimal reproduction of the problem with instructions**

1. Write worker code:
```typescript
export default class Api {
  foo() {
    return null;
  }
}
```
2. Write a client that calls this API:
```typescript
const client = new SmartClient();
client.connect();
const api = client.createProxyTaskService()
```
3. Call the API from client:
```typescript
console.log(await api.foo()) // it displays Object({ })
```

**What is the motivation / use case for changing the behavior?**

If I explicitly return null or undefined in API it is mainly because I wanted to do so (for example explicitly tell that there is nothing). Front code would be more complex to check if the result is nothing or not. Currently we can't differentiate wanted empty object from wanted null value:
```typescript
export default class Api {
  foo(bar?: string) {
    return bar ? JSON.parse(string) : null;
  }
```

**Please tell us about your environment:**

* **Operating System:**
* **Node Version:** v10.8.0
* **Npm Version:** 6.2.0
* **@zetapush/\* Version:** 0.33.2

**Comment by ghoullier** [comment]
createProxyService method catch a default value on cloud service function result: https://github.com/zetapush/zetapush/blob/master/packages/client/src/client/helper.js#L499

**Comment by aurelien-baudet** [comment]
Should we really provide this default value ?
Do we provide a way to configure that default value or is it the responsibility of the developer ?
It seems that we won't be able to differentiate undefined from null if we remove that default value. Is it something we should handle correctly ?

**Comment by ghoullier** [comment]
I agree with you. This behavior break the contract between worker and the front end application.
We shouldn't manage a default return value.
